auth-web/src/components/datatable/BaseVDataTable.vue

```diff
@@ -138,63 +139,42 @@ export default defineComponent({
       filtering: false,
       headers: _.cloneDeep(props.setHeaders),
       sortedItems: [...props.setItems],
-      visibleItems: [],
-      tableDataOptions: props.setTableDataOptions
+      tableDataOptions: props.setTableDataOptions,
+      visibleItems: []
     }) as unknown) as BaseTableStateI
 
-    const start = ref(0)
-    const rowHeight = ref(75)
-    const perPage = 10
-    const timeout = ref(null)
-    const vm = getCurrentInstance().proxy
-    const regTable = ref(null)
-
-    const onScroll = (e) => {
-      timeout && clearTimeout(timeout)
-      timeout.value = setTimeout(() => {
-        const scrollTop = e.target.scrollTop
-        const rowsInView = Math.ceil(e.target.clientHeight / rowHeight.value)
-        let startRow = Math.floor(scrollTop / rowHeight.value)
-        const endRow = startRow + rowsInView
-
-        // Handle scrolling up
-        if (startRow < start.value) {
-          start.value = startRow
-        }
-
-        // Handle scrolling down
-        if (endRow > start.value + perPage) {
-          start.value = endRow - perPage
+    const currentPage = ref(1)
+    const perPage = ref(50)
+    const firstItem = ref(null) // first item in table
+    const reachedEnd = ref(false)
+
+    const getNext = _.throttle(() => {
+      // if not loading and reg history exists
+      if (!props.loading && !reachedEnd.value && state.sortedItems.length > state.visibleItems.length) {
+        currentPage.value++
```

Comment:
Maybe put in a check.. if the scroll bar is at the bottom, keep loading new rows?